src/metaschema/oscal_poam_metaschema.xml

```diff
@@ -57,6 +57,68 @@
       </model>
    </define-assembly>
    
+   <define-assembly name="poam-items">
+      <formal-name>POA&amp;M Items</formal-name>
+      <description>This identifies initial and residual risks, deviations, and disposition.</description>
+      <model>
+         <field ref="title"  min-occurs="1" max-occurs="1" />
+         <field ref="description"  min-occurs="1" max-occurs="1" />
+         <field ref="prop"  min-occurs="0" max-occurs="unbounded">
+            <description>Provided as means of extending the OSCAL syntax.</description>
+            <group-as name="properties" in-json="ARRAY"/>
+         </field>
+         <assembly ref="annotation" min-occurs="0" max-occurs="unbounded">
+            <description>Provided as means of extending the OSCAL syntax.</description>
+            <group-as name="annotations" in-json="ARRAY"/>
+         </assembly>
+         <field ref="start" min-occurs="1" max-occurs="1" >
+            <description>Date/time stamp identifying the start of the evidence collection reflected in these results.</description>
+         </field>
+         <field ref="end" min-occurs="1" max-occurs="1">
+            <description>Date/time stamp identifying the end of the evidence collection reflected in these results. In a continuous motoring scenario, this may contain the same value as start if appropriate.</description>
```

Comment:
motoring -> monitoring

I think it would be more reasonable to make this optional (`min-occurs="0"`) to indicate continuous monitoring.

Reply:
@bradh agreed! Good suggestion, and consistent with other POA&M tweaks we are making.

Reply:
+1.

You still need to fix the "motoring" typo.